src/ontology/mondo-edit.obo

```diff
@@ -532365,6 +532365,35 @@ xref: DOID:0081452 {source="MONDO:equivalentTo"}
 is_a: MONDO:0004095 {source="DOID:0081452"} ! B-cell neoplasm
 is_a: MONDO:0005062 {source="https://orcid.org/0000-0002-4142-7153"} ! lymphoma
 
+[Term]
+id: MONDO:0968986
+name: chaotic conus spinal cord lipoma
+def: "A rare dysraphic spinal cord lipoma characterized by the lipomatous mass extending ventrally to the dorsal root entry zone, indicating a more severe malformation of the spinal cord. The diagnosis can be suggested on imaging but usually confirmed during surgery." [Orphanet:645285]
+xref: Orphanet:645285 {source="MONDO:equivalentTo"}
+is_a: MONDO:0958350 {source="Orphanet:645285"} ! conus spinal cord lipoma
```

Comment:
did we officially decide if we wanted to add our ORCID to ORDO only supported superclasses, so we know that we reviewed them and agree with them? Or do we no longer want to do this now that we did the strategic reclassification?

Reply:
I forgot about it! :-) I had a moment of "I should add my orcid" then moved on without doing it... I will add it for all the Orphanet source that I reviewed. Thank you for catching this!